src/frontend/themes.scss

```diff
@@ -346,6 +345,13 @@ body.nord-light {
   --gamecard-title-color: #ccdaf8cc;
   --install-button: var(--primary);
   --status-success-hover: #5e81ac;
+  --text-log: var(--status-warning);
+  --tour-popup-background: var(--input-background);
+  --tour-bullet-active: var(--accent);
+  --tour-bullet-inactive: var(--accent-overlay);
+  --tour-popup-title: var(--text-title);
+  --tour-popup-skip-color: var(--text-secondary);
+  --tour-popup-skip-color-hover: var(--text-title);
```

Comment:
The tour was practically unusable for nord-light:

![image](https://github.com/user-attachments/assets/5d303afe-49fc-468c-a1c1-3b3960de4dc3)

and now:
![image](https://github.com/user-attachments/assets/23d83597-2552-4ccb-bc4a-5f2c691401d2)